jetcd-core/src/main/java/io/etcd/jetcd/ClientConnectionManager.java

```diff
@@ -330,35 +331,84 @@ public void onClose(Status status, Metadata trailers) {
   public <S, T> CompletableFuture<T> execute(
       Callable<ListenableFuture<S>> task,
       Function<S, T> resultConvert) {
-    return execute(task, resultConvert, Util::isRetryable);
+    return execute(task,resultConvert, Util::isRetryable);
```

Comment:
this change is unrelated to the PR

Reply:
here,it a codestyle conflict via my local workspace. I withdraw this commit